packages/@magic-ext/web3modal-ethers5/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@magic-ext/web3modal-ethers5",
+  "version": "0.1.0",
+  "description": "magic web3modal ethers 5 extension",
+  "author": "Magic <team@magic.link> (https://magic.link/)",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/magiclabs/magic-js"
+  },
+  "files": [
+    "dist"
+  ],
+  "target": "neutral",
+  "cdnGlobalName": "MagicWeb3ModalExtension",
+  "main": "./dist/cjs/index.js",
+  "module": "./dist/es/index.js",
+  "types": "./dist/types/index.d.ts",
+  "jsdelivr": "./dist/extension.js",
+  "exports": {
+    "import": "./dist/es/index.mjs",
+    "types": "./dist/types/index.d.ts",
+    "require": "./dist/cjs/index.js"
+  },
+  "externals": {
+    "include": [
+      "@magic-sdk/commons"
+    ]
+  },
+  "devDependencies": {
+    "@magic-sdk/commons": "^24.0.2",
+    "@magic-sdk/types": "24.0.2-canary.742.9175925480.0"
```

Comment:
is this canary version needed here?

Reply:
Because `@magic-sdk/types` is a separate package, which is getting updated types that this extension is using, we need `@magic-sdk/types` to be published. So I think yes we need to reference a canary version at least at the first publish, then we can update the extension to use a non-canary